code/game/objects/items/devices/scanners.dm

```diff
@@ -241,12 +241,13 @@ SLIME SCANNER
 		msgs += "<span class='warning'>Subject appears to have [H.getCloneLoss() > 30 ? "severe" : "minor"] cellular damage.</span>"
 
 	// Brain.
-	if(H.get_int_organ(/obj/item/organ/internal/brain))
-		if(H.getBrainLoss() >= 100)
+	var/obj/item/organ/internal/brain = H.get_int_organ(/obj/item/organ/internal/brain)
+	if(brain)
+		if(brain.damage >= (brain.max_damage * 10 / 12)) // 100
```

Comment:
The reason is it's done like this is because a normal brain has 120 hp (one of the arbitrary amounts of all time). This was done so the ratios stay the same for normal brains, but do adjust for a lower max amount of brain hp